src/app/zap-templates/zcl/data-model/chip/wifi-network-diagnostics-cluster.xml

```diff
@@ -33,7 +33,24 @@ limitations under the License.
     <item name="802.11n" value="0x03"/>
     <item name="802.11ac" value="0x04"/>
     <item name="802.11ax" value="0x05"/>    
+  </enum>
+  <enum name="AssociationFailureCauseType" type="ENUM8">
+    <cluster code="0x0036"/>
+    <item name="Unknown" value="0x00"/>
+    <item name="AssociationFailed" value="0x01"/>
+    <item name="AuthenticationFailed" value="0x02"/>
+    <item name="SsidNotFound" value="0x03"/>
+  </enum>
+  <enum name="ConnectionStatusType" type="ENUM8">
+    <cluster code="0x0036"/>
+    <item name="Connected" value="0x00"/>
+    <item name="NotConnected" value="0x01"/>
   </enum>  
+  <struct name="AssociationFailureStruct">
```

Comment:
Where is this struct definition coming from?  I don't think this should be here....

Reply:
Yes, it is removed, my pdf version seems out of date